### PR TITLE
The DHCP shim configuration should include type: Raw 

### PR DESCRIPTION
**Pull request by dougbtv**
This adds the `type: Raw` field, as well as indents the curly braces.

### CODE REVIEW

**Comment by mikemckiernan** [comment]
Doug's update is to the code block at https://deploy-preview-30167--osdocs.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-bridge.html#nw-multus-dhcp_configuring-bridge

Applies to 4.8 through 4.5.  Please add milestone Next-Release.

@openshift/team-documentation PTAL.

**Review by kalexand-rh — COMMENTED** [review]
This LGTM, but it still needs QE approval before we can merge it.

**Comment by mikemckiernan** [comment]
/assign @weliang1 

Weibin, the team tracking XLS has you listed as the QE contact for Multus.  If you are not the correct QE contact, can you reassign it?  Thank you.

**Comment by jboxman** [comment]
@dougbtv thanks for this!

**Review by s1061123 — APPROVED** [review]
It looks good to me, too! Thanks!

**Comment by bmcelvee** [comment]
Reviews complete - merging + cherry picking. :)

**Pull request merged**
